Content.Shared/Armable/ArmableComponent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Robust.Shared.GameStates;
+
+namespace Content.Shared.Armable;
```

Comment:
You need a newline here

```suggestion
namespace Content.Shared.Armable;

```